parser/tests/integration/test_parser_integration.py

```diff
@@ -0,0 +1,95 @@
+import pytest
+import pathlib
+from openfoodfacts_taxonomy_parser import parser
+
+# taxonomy in text format : test.txt
+TEST_TAXONOMY_TXT = str(pathlib.Path(__file__).parent.parent / "data" / "test")
+
+
+@pytest.fixture
+def new_session():
+    x = parser.Parser()
+    # delete all the nodes and relations in the database
+    query="MATCH (n) DETACH DELETE n"
+    x.session.run(query)
+    return x
+
+
+def test_calling(new_session):
+    x=new_session
```

Comment:
If you change your feature to `auto_use=True`, it would be 
```suggestion
def test_calling():
    x = parser.Parser()
```

Reply:
but I would also change x to test_parser or taxonomy_parser

Reply:
and you might have a session variable `session = test_parser.session` to avoid long line thereafter